pkg/kubelet/apis/kubeletconfig/types.go

```diff
@@ -58,6 +58,9 @@ type KubeletConfiguration struct {
 	// podManifestPath is the path to the directory containing pod manifests to
 	// run, or the path to a single manifest file
 	PodManifestPath string
+	// bootstrapCheckpointPath is the path to the directory containing pod checkpoints to
+	// run on restore
+	BootstrapCheckpointPath string
```

Comment:
BTW, removing this from the configuration may make node e2e testing harder, just FYI.

Reply:
Yup, and just to add some context:
1. Removing it from the configuration for now doesn't prevent you from adding it back later, once you're more confident in the stability of your API and behavior (e.g. no longer an alpha feature).
2. If you do want an alpha feature in the API, these are the guidelines regarding feature gating it properly: https://github.com/kubernetes/community/blob/master/contributors/devel/api_changes.md#alpha-field-in-existing-api-version

Reply:
>  He has removed all Alpha features from the kubeletconfig, unless they are feature-gated. 

@yujuhong This is patently untrue.  

NodeLabels, VolumePluginDir, CPUManagerPolicy, LockFilePath are all alpha labeled features that exist in the kubeletconfig.  The usage of componentconfig is also alpha, from what I've seen.

Reply:
I can do whatever, but your statement doesn't reconcile with what is there.

Reply:
@timothysc `CPUManagerPolicy` is feature gated, which is why it remains.
`NodeLabels`, `VolumePluginDir`, and `LockFilePath` were all moved from `KubeletConfiguration` to `KubeletFlags` in #55562, because they are alpha but lack feature gates.
This is in preparation for moving `KubeletConfiguration` to beta.
Please feature gate your field if you want to leave it in `KubeletConfiguration` or otherwise move it to `KubeletFlags`.
If you have more questions, feel free to ping me on Slack.

Reply:
Fixed, following #55562 as reference.